devicon.json

```diff
@@ -2484,6 +2484,20 @@
         "color": "#F18803",
         "aliases": []
     },
+    {
+        "name": "perl",
+        "tags": [
+            "programming",
+            "language"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "plain"
+            ]
+        },
+        "color": "#212177"
```

Comment:
Add this attribute:
```
,
"aliases": []
```
While we don't enforce the aliases, right now the script still look for an aliases attribute.

Reply:
```suggestion
        "color": "#212177",
        "aliases": []
```